paper-dropdown-menu.html

```diff
@@ -109,7 +109,7 @@
           invalid="[[invalid]]"
           readonly
           disabled="[[disabled]]"
-          value="[[selectedItemLabel]]"
```

Comment:
so is this still being used anywhere?

Reply:
Not actively in the element, but it is exposed as public notifying read-only property, so would be a breaking change to remove it :/